be/src/runtime/fragment_mgr.cpp

```diff
@@ -220,6 +222,39 @@ std::string FragmentExecState::to_http_path(const std::string& file_name) {
     return url.str();
 }
 
+Status FragmentExecState::report_exec_status(const TReportExecStatusParams& params) {
+    TReportExecStatusResult res;
+    Status st;
+
+    FrontendServiceConnection coord(_exec_env->frontend_client_cache(), _coord_addr, &st);
+    if (!st.ok()) {
+        std::stringstream ss;
+        ss << "couldn't get a client for " << _coord_addr << ", error: " << st;
+        LOG(WARNING) << ss.str();
+        return Status::InternalError(ss.str());
```

Comment:
there is no `_executor.cancel()` when creating connection before.
is this a bug?